src/components/common/helpers/compareVersion.ts

```diff
@@ -24,3 +24,7 @@ export function isVersionLessThanOrEqualToTarget(version: string, target: number
     }
     return false;
 }
+
+export function isChartRefIdLessThanOrEqualToTarget(id: number, target: number): boolean {
```

Comment:
This function has nothing to do with chart. So function name could be "isLessThanOrEqualToTarget"

Reply:
have changed the function name